lib/geofence_foreground_service_method_channel.dart

```diff
@@ -193,8 +187,7 @@ class MethodChannelGeofenceForegroundService
 
     try {
       areAllAreasRemoved =
-          await foregroundChannel.invokeMethod<bool?>('removeAllGeoFences') ??
-              false;
+          await foregroundChannel.invokeMethod<bool?>('removeAllGeoFences') ?? false;
```

Comment:
Can we replace the hardcoded values in this file with the new ones created in the ApiMethods class?

Reply:
Added ApiMethods class in dart aswell and replaced the method invocations to use the constants.
Also, I've modified the method invocations to directly return the value inside the "try" blocks, to make the code more readable.